.github/workflows/conda-build.yml

```diff
@@ -32,7 +32,9 @@ jobs:
         python-version: 3.11.x
     - name: Pull Docker image
       run: |
-        docker image pull public.ecr.aws/codebuild/local-builds:latest
+        docker image pull public.ecr.aws/codebuild/amazonlinux-x86_64-standard:5.0
```

Comment:
Just for my understanding, why didn't `local-builds:latest` work? In fact, with all the recently applied fixes, would we expect it to now work?